packages/hardhat/contracts/FundingVault.sol

```diff
@@ -56,13 +57,34 @@ contract FundingVault{
     uint256 private blockLimit;
     uint256 private minFundingAmount; //The minimum amount of ETH required in the contract to enable withdrawal.
```

Comment:
`minFundingAmount` should be immutable.

Reply:
Fixed: minFundingAmount is immutable in latest commit